packages/flutter/lib/src/cupertino/sheet.dart

```diff
@@ -140,16 +140,22 @@ final Animatable<double> _kScaleTween = Tween<double>(begin: 1.0, end: 1.0 - _kS
 ///  * <https://developer.apple.com/design/human-interface-guidelines/sheets>
 Future<T?> showCupertinoSheet<T>({
   required BuildContext context,
-  required WidgetBuilder pageBuilder,
+  @Deprecated(
+    'Use builder instead. '
+    'This feature was deprecated after v3.33.0-0.2.pre.',
+  )
+  WidgetBuilder? pageBuilder,
+  WidgetBuilder? builder,
   bool useNestedNavigation = false,
   bool enableDrag = true,
 }) {
-  final WidgetBuilder builder;
+  assert(pageBuilder != null || builder != null);
+  final WidgetBuilder widgetBuilder;
   final GlobalKey<NavigatorState> nestedNavigatorKey = GlobalKey<NavigatorState>();
   if (!useNestedNavigation) {
-    builder = pageBuilder;
+    widgetBuilder = (pageBuilder ?? builder)!;
```

Comment:
Nit: should flip the priority so builder is used before pageBuilder, if both are non-null. More for the sake of reading the intent in the code.